recipes/neutronics-material-maker/meta.yaml

```diff
@@ -0,0 +1,46 @@
+
+{% set name = "neutronics_material_maker" %}
+{% set version = "0.1.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 43f89bc94ca76bd45fe26cb59b1a17c801ccf0db1578d304589f050175070def
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - openmc
+    - CoolProp
```

Comment:
packages names should be lowercase

Reply:
Thanks for that, sorry for not spotting it earlier.